apps/test/unit/code-studio/searchAssetsTest.js

```diff
@@ -0,0 +1,76 @@
+/** @file Test of searchAssetsTest.js. */
+
+var assert = require('assert');
+import {searchAssets} from '@cdo/apps/code-studio/assets/searchAssets';
+var animationLibrary = require('@cdo/apps/gamelab/animationLibrary.json');
+var soundLibrary = require('@cdo/apps/code-studio/soundLibrary.json');
+
+describe('search assets from animation library', function () {
+  let maxResults = 5;
+  let pageCount = 0;
```

Comment:
I should be able to run tests in any order and have things work. Right now, if I run the last test here first, maxResults/pageCount will suddenly be different.

You have a few options
- Define them in each test
- Define them as consts here, and then in tests where you need to change them use different variables.
- have a before method that resets these (I probably like this option least).